js/cmd-resume.js

```diff
@@ -0,0 +1,744 @@
+				self.commands.education = {
+					title: "Education",
+					description: "education history",
+					data: self.data.education,
+					type: self.commandProcessor.array,
+					handlers:{
+						organisation: function(value){
+							return value.institution;
+						},
+						title: function(value){
+							return getFullDegree(value.studyType, value.area);
+						},
+						date: function(value){
+							return getDate(value.startDate, value.endDate);
+						}
+					}
+				};
+			}
+
+			if (self.data.work && self.data.work.length > 0){
+				self.commands.employment = {
+					title: "Employment",
+					description: "employment history",
+					data: self.data.work,
+					type: self.commandProcessor.array,
+					handlers:{
+						organisation: function(value){
+							return value.company;
+						},
+						title: function(value){
+							return value.position;
+						},
+						date: function(value){
+							return getDate(value.startDate, value.endDate);
+						}
+					}
+				};
+			}
+
+			if (self.data.volunteer && self.data.volunteer.length > 0){
+				self.commands.volunteering = {
+					title: "Volunteering",
+					description: "volunteering history",
+					type: self.commandProcessor.array,
+					data: self.data.volunteer,
+					handlers:{
+						organisation: function(value){
+							return value.organization;
+						},
+						title: function(value){
+							return value.position;
+						},
+						date: function(value){
+							return getDate(value.startDate, value.endDate);
+						}
+					}
+				};
+			}
+
+			if (self.data.awards && self.data.awards.length > 0){
+				self.commands.awards = {
+					title: "Awards",
+					description: "awards obtained",
+					type: self.commandProcessor.array,
+					data: self.data.awards,
+					handlers:{
+						organisation: function(value){
+							return value.awarder;
+						},
+						title: function(value){
+							return value.title;
+						},
+						date: function(value){
+							return value.date;
+						}
+					}
+				};
+			}
+
+			if (self.data.publications && self.data.publications.length > 0){
+				self.commands.publications = {
+					title: "Publications",
+					description: "publications produced",
+					type: self.commandProcessor.array,
+					data: self.data.publications,
+					handlers:{
+						organisation: function(value){
+							return value.publisher;
+						},
+						title: function(value){
+							return value.name;
+						},
+						date: function(value){
+							return value.releaseDate;
+						}
+					}
+				};
+			}
+
+			if (self.data.languages && self.data.languages.length > 0){
+				self.commands.languages = {
+					title: "Languages",
+					description: "languages",
+					type: self.commandProcessor.array,
+					data: self.data.languages,
+					handlers:{
+						organisation: function(value){
+							return value.language;
+						},
+						title: function(value){
+							return value.fluency;
+						},
+						date: function(value){
+							return "";
```

Comment:
Would make more sense to not have this function there to begin with. Issue opened.